content/docs/dvclive/api-reference/init.md

```diff
@@ -45,11 +43,6 @@ all existing `.tsv` files, `{path}.json` and `{path.html}`.
   call, DVCLive will generate a _metrics summary_ (usable by `dvc metrics`). The
   _summary_ will be located at `{path}.json`.
 
-- `html` (`True` by default) - works only when DVCLive is used alongside DVC. If
```

Comment:
There's still a note above about cleanup removing `{path.html}`. I think this is still technically correct, but it seems a bit confusing without any other mention of html files.